README.adoc

```diff
@@ -53,170 +53,122 @@ include::{common-includes}/gitclone.adoc[]
+A health report will be generated automatically for all services that enable MicroProfile Health. This
+feature has already been enabled for you in the `src/main/liberty/config/server.xml` file:
+
+[source, xml, role="no_copy"]
+----
+<feature>mpHealth-1.0</feature>
+----
+
+All applications must provide an implementation of the `HealthCheck` interface, which will be used to
+verify the healthiness of the services.
+
 
 // =================================================================================================
 // Adding a health check to the system service
 // =================================================================================================
 
-=== Adding a health check to the system service
+=== Adding a health check to the `system` service
+
+Before you proceed, navigate to the `start` directory where you will find the starting project:
+
+```
+cd ../start
+```
 
-Create health check class for the `system` service in the
-`src/main/java/io/openliberty/guides/system/SystemHealth.java` file:
+Implement the `HealthCheck` interface in the `src/main/java/io/openliberty/guides/system/SystemHealth.java` file:
 
 [source, java, indent=0]
 ----
 include::finish/src/main/java/io/openliberty/guides/system/SystemHealth.java[tags=SystemHealth]
 ----
 
-The `org.eclipse.microprofile` package contains the required MicroProfile Health Check classes.
-
-The `@Health` annotation indicates that this class reports the state of
-`system` service through `http://localhost:9080/health` URL. The `@ApplicationScoped` annotation is
-required because the health check service must stay active within the lifetime of the application
-to report the state.
+The `@Health` annotation indicates that this particular bean implements the `HealthCheck` interface.
+By pairing this annotation with the `ApplicationScoped` context from the CDI API, the bean is discovered
+automatically when the `http://localhost:9080/health` endpoint receives a request.
 
-This class implements the `HealthCheck` interface from the MicroProfile Health API. The class also implements the `call()`
-method from the interface to report the state of the service by returning a `HealthCheckResponse` object. The `system`
-service self-checks. It reads the system property of the host to report a healthy state.
-If the `wlp.server.name` system property is `defaultServer`, the state is `UP`. Otherwise, the state is `DOWN`.
+The `call()` method is used to return the health status of a particular service. In this case, you are simply
+checking if the server name is `defaultServer` and returning `UP` if it is, and `DOWN` otherwise.
+The `HealthCheckResponse.named()` method is used to indicate what service the health check is done for.
+Overal, this is a very simple implementation of the `call()` method. In a real development environment,
+you would want to orchestrate much more meaningful health checks.
```

Comment:
Typo with 'Overal'.

Reply:
good catch, fixed

Reply:
Also, I missed this -- It wouldn't be "real development environment" but "real world usage" or equivalent.